### PR TITLE
fix(playground): add `tabIndex` to json editors

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Related to https://github.com/camunda/team-hto/issues/385

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Added the suggested fix from https://discuss.codemirror.net/t/is-there-a-way-to-configure-the-tabindex-of-the-textbox-without-hacking-the-source/5111/2

File: packages/form-js-playground/src/components/PlaygroundRoot.js [hunk 1]
```diff
@@ -91,14 +91,14 @@ export function PlaygroundRoot(props) {
   useEffect(() => {
     const dataEditor = dataEditorRef.current = new JSONEditor({
       value: toString(data),
-      contentAttributes: { 'aria-label': 'Form Input' },
+      contentAttributes: { 'aria-label': 'Form Input', tabIndex: 0 },
```
Comment (vsgoulart) [inline packages/form-js-playground/src/components/PlaygroundRoot.js:94]: isn't it better to use `undefined` as default?

File: packages/form-js-playground/test/spec/Playground.spec.js [hunk 2]
```diff
@@ -790,6 +791,60 @@ describe('playground', function() {
 
   });
 
+
+  describe('a11y', function() {
+
+    it('should have no violations', async function() {
+
+      // given
+      this.timeout(5000);
```
Comment (vsgoulart) [inline packages/form-js-playground/test/spec/Playground.spec.js:800]: Why is the timeout necessary?

**Review by vsgoulart — APPROVED** [review]

File: packages/form-js-playground/test/spec/Playground.spec.js (hunk 2 above)
Comment (pinussilvestrus) [reply packages/form-js-playground/test/spec/Playground.spec.js:800]: `axe` takes a while to run its validations, leading `karma` to run out of time sometimes (flaky).

File: packages/form-js-playground/src/components/PlaygroundRoot.js (hunk 1 above)
Comment (pinussilvestrus) [reply packages/form-js-playground/src/components/PlaygroundRoot.js:94]: I think the creators of Code Mirror suggested a non-undefined value as the parent `.cm-scroller` container is `-1`. This is not configurable so we need to set the `cm-content` container explicitly.

image

This is where the a11y error comes from in the first place

image

**Pull request merged**